src/api/handlers/technologies.js

```diff
@@ -0,0 +1,15 @@
+import Axios from 'axios';
+
+const options = {
+  method: 'GET',
+  headers: {
+    'x-api-key' : 'YsO9MGt0DwUu2PDTQ4y4TrhRtnJlYFzYWa1NW92PmvDFqcyDGLiacSNDWape'
+  }
+};
+
+const getAllTechnologies = async () => {
+  const response = await Axios.get('https://658m0jk5l3.execute-api.us-east-1.amazonaws.com/prod/technology/get', options);
+  return response;
+};
+
+export default getAllTechnologies;
```

Comment:
I think we need to incorporate a try-catch in each handler. Can't just return response - need to return specific data and a status code